### ISSUE TITLE
getting CropPol indexed by GloBI

### ISSUE THREAD

**Issue by jhpoelen** [ibartomeus/OBservData#5]
Hi @ibartomeus -

Thanks for openly sharing (and keeping up-to-date) your CropPol database ! 

I was wondering whether you'd be interested in getting your data indexed by GloBI? 

If so, please holler and we can share some ideas on how to setup the index configuration to make sure that your CropPol data (and updates) are indexed automatically. 

thx,
-jorrit

fyi @zedomel @deboradrucker @seltmann

**Comment by jhpoelen** [comment]
related to https://github.com/globalbioticinteractions/globalbioticinteractions/issues/830 .

**Comment by ibartomeus** [comment]
Hi Jorrit,

Sure, this would be great to do, but we will probably be slow at implementing new things, as the project is right now underfunded.

Best,

**Comment by ibartomeus** [comment]
I guess this can be easily done by merging the tables. Is this the only thing you would need?

**Comment by jhpoelen** [comment]
> I guess this can be easily done by merging the tables. Is this the only thing you would need?

Yes, with a file containing the merged records from the three tables sampling, field, ownership, I can apply a index configuration and make your data findable through GloBI. Perhaps most importantly is to make sure that the original data contributors (the "owners" ?) are attributed in some kind of citation string / citation tables.

Thanks for taking the time to respond, and looking forward to adding CropPol to GloBI's search index.

Please let me know if you need more info from me to proceed.

**Comment by ibartomeus** [comment]
ok, expect this to be done after christmas beak. I'll let you know here.

**Comment by jhpoelen** [comment]
@ibartomeus sounds wonderful! I much appreciate your willingness to tweak your data products to make it easier for GloBI (and others) to re-use your work.
